src/main/java/com/diffmin/GumtreeSpoonAstDiff.java

```diff
@@ -16,6 +16,12 @@
  * taken from.
  */
 public class GumtreeSpoonAstDiff {
+
+    /** Override constructor to prevent instantiating of this class (RSPEC-1118). */
+    private GumtreeSpoonAstDiff() {
```

Comment:
Although this constructor will decrease the coverage, should we care writing a test for it? This class has to be made as a library so we can just reuse it.

Reply:
No need to write a test for a private constructor. In fact, that's inconveniently hard (you'd need to change the visibility with reflection or something like that).